packages/web-react/src/components/Modal/demo/ModalScrollingLongContent.tsx

```diff
@@ -16,6 +71,24 @@ const ModalScrollingLongContent = () => {
   const handleModalScrollingInsideClose = () => setModalScrollingInsideOpen(false);
   const handleModalCustomHeightClose = () => setModalCustomHeightOpen(false);
 
+  const handleMaxHeightChange = (event: ChangeEvent<HTMLInputElement>, breakpoint: BreakpointToken) => {
```

Comment:
It's interesting the change event in React is being fired _during_ the interaction, not _after_ it like in the `web` demo. Just saying 💁🏻‍♂️